app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,6 @@
+class FoodsController < ApplicationController
+  def index
+    @food = params[:q]
```

Comment:
I don't think you're using this instance var anywhere except on line 4 so you could probably use a local variable instead, or even remove this line altogether and pass the param directly in line 4 like this:
`@foods = FoodFacade.foods(params[:q])`